string name
int32 tag_id
---
bool success
HighValueTarget info
